From e53e416e2e03abce4c1c097e5d7057f47d1978dc Mon Sep 17 00:00:00 2001
From: Ian Cordasco <graffatcolmingov@gmail.com>
Date: Sat, 7 Nov 2015 12:14:57 -0600
Subject: [PATCH] Exclude tests.integration from find_packages

Closes #82
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e60d8f9..54b1551 100644
--- a/setup.py
+++ b/setup.py
@@ -3,7 +3,7 @@
 import sys
 from setuptools import setup, find_packages
 
-packages = find_packages(exclude=['tests'])
+packages = find_packages(exclude=['tests', 'tests.integration'])
 requires = ['requests >= 2.0']
 
 __version__ = ''
