From: Manuel Rueger <gentoo@rueg.eu>
Date: Sat Sep  8 20:06:52 CEST 2012
Subject: Include missing classes

--- source/soundmanager/js/MusicSound.cpp.old	2012-09-08 19:09:45.640423091 +0200
+++ source/soundmanager/js/MusicSound.cpp	2012-09-08 19:28:43.442346286 +0200
@@ -26,6 +26,9 @@
 #include "ps/Filesystem.h"
 #include "soundmanager/SoundManager.h"
 
+#include <iostream>
+#include <sstream>
+#include <string>
 
 JMusicSound::JMusicSound(const VfsPath& pathname) : m_FileName(pathname)
 {
--- source/soundmanager/js/SoundPlayer.cpp.old	2012-09-08 19:41:14.733295572 +0200
+++ source/soundmanager/js/SoundPlayer.cpp	2012-09-08 19:41:20.042295213 +0200
@@ -24,6 +24,9 @@
 #include "ps/Filesystem.h"
 #include "soundmanager/SoundManager.h"
 
+#include <iostream>
+#include <sstream>
+#include <string>
 
 JSoundPlayer::JSoundPlayer()
 {
