http://trac.wildfiregames.com/ticket/1664

Index: /ps/trunk/source/soundmanager/js/SoundPlayer.cpp
===================================================================
--- /ps/trunk/source/soundmanager/js/SoundPlayer.cpp	(revision 12654)
+++ /ps/trunk/source/soundmanager/js/SoundPlayer.cpp	(revision 12655)
@@ -24,4 +24,6 @@
 #include "ps/Filesystem.h"
 #include "soundmanager/SoundManager.h"
+
+#include <sstream>
 
 
Index: /ps/trunk/source/soundmanager/js/MusicSound.cpp
===================================================================
--- /ps/trunk/source/soundmanager/js/MusicSound.cpp	(revision 12654)
+++ /ps/trunk/source/soundmanager/js/MusicSound.cpp	(revision 12655)
@@ -26,4 +26,6 @@
 #include "ps/Filesystem.h"
 #include "soundmanager/SoundManager.h"
+
+#include <sstream>
 
 
