--- main.cc.old	2009-04-01 13:56:18.000000000 +0200
+++ main.cc	2012-05-24 04:16:37.000000000 +0200
@@ -1544,7 +1544,7 @@
 
 #else 
 
-    libdir = "/usr/local/MobileSim";
+    libdir = "/usr/share/MobileSim";
     stg_print_msg("MobileSim: Expecting supporting resources to be installed in the default location: \"%s\".", libdir);
 #endif
 
--- config/MobileSim.desktop.old	2012-05-24 04:22:03.142344992 +0200
+++ config/MobileSim.desktop	2012-09-09 18:30:09.817406764 +0200
@@ -1,11 +1,10 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-Encoding=UTF-8
 Name=MobileSim
 Comment=Simulate MobileRobots/ActivMedia robots in their environment, for use with ARIA
-Icon=/usr/local/MobileSim/icon.png
-TryExec=/usr/local/MobileSim/MobileSim
-Path=/usr/local/bin
-Exec=/usr/local/bin/MobileSim
-Categories=Application;Science;
+Icon=/usr/share/MobileSim/icon.png
+TryExec=/usr/MobileSim
+Path=/usr/bin
+Exec=/usr/bin/MobileSim
+Categories=Education;Science;
--- CrashHandler.cc.old	2012-05-24 04:25:15.328329716 +0200
+++ CrashHandler.cc	2012-05-24 04:25:44.517327399 +0200
@@ -54,7 +54,7 @@
     const char *gdbhelper = "gdbhelper";
     struct stat statbuf;
     if(stat(gdbhelper, &statbuf) != 0) {
-        gdbhelper = "/usr/local/MobileSim/gdbhelper";
+        gdbhelper = "/usr/share/MobileSim/gdbhelper";
         if(stat(gdbhelper, &statbuf) != 0) {
             gdbhelper = NULL;
             stg_print_error("Couldn't find gdbhelper in current directory or as /usr/local/MobileSim/gdbhelper, can't save debugging information.");
@@ -63,7 +63,7 @@
 
     const char *progname = mobilesim_argv[0];
     if(stat(progname, &statbuf) != 0) {
-        progname = "/usr/local/MobileSim/MobileSim";
+        progname = "/usr/bin/MobileSim";
         if(stat(progname, &statbuf) != 0) {
             stg_print_error("Can't find MobileSim program (tried %s and %s), Can't save debugging information or restart program.", mobilesim_argv[0], progname);
             return;
--- StageInterface.cc.old	2012-05-24 04:29:20.260310253 +0200
+++ StageInterface.cc	2012-05-24 04:29:30.952309401 +0200
@@ -1460,7 +1460,7 @@
     // previous versions of MobileSim interpreted as automatically being
     // reflectors.
     int val = 1;
-    char* endTag = strrchr(type, '.');
+    char* endTag = (char*) strrchr(type, '.');
     if(   strcmp( type, "Sim.Reflector") == 0 
       || strcmp(  type, "Reflector") == 0
       || (endTag && strcmp(endTag, ".Reflect") == 0)
