--- Makefile.old	2009-04-06 10:13:27.000000000 +0200
+++ Makefile	2012-05-24 04:27:56.907316877 +0200
@@ -39,6 +39,7 @@
 SHELL=$(warning at rule   $@.   Modified prerequisites are: [$?]    All prerequisites are: [$^])$(REAL_SHELL)
 endif
 
+MOBILESIM_RELEASE:=1
 
 all: MobileSim
 
@@ -66,7 +67,6 @@
 #### Different options for debug or release modes
 ifdef MOBILESIM_RELEASE
 
-CFLAGS+=-O2
 STAGE_CONFIGURE_ARGS:=--disable-debug --enable-optimize=2 --enable-tests
 
 else	 #MOBILESIM_RELEASE
@@ -231,14 +231,14 @@
 
 # For installation:
 ifndef prefix
-prefix:=$(DESTDIR)/usr/local/MobileSim
+prefix:=$(DESTDIR)/usr/share/MobileSim
 endif
 
-bindir:=$(prefix)
-docdir:=$(prefix)
+bindir:=$(DESTDIR)/usr/bin
+docdir:=$(DESTDIR)/usr/share/doc/mobilesim-$(VERSION)
 confdir:=$(prefix)
 ourlibdir:=$(prefix)/lib
-sysbindir:=$(DESTDIR)/usr/local/bin
+sysbindir:=$(DESTDIR)/usr/bin
 
 # Bypases DESTDIR:
 ifndef installed_bindir
@@ -386,7 +386,7 @@
 
 install:
 	install -d $(bindir) $(docdir) $(confdir) $(DESTDIR)/usr/share/applications $(sysbindir) 
-	install -s -m 755 MobileSim $(bindir)/MobileSim
+	install -m 755 MobileSim $(bindir)/MobileSim
 	install -m 644 icon.png $(confdir)/icon.png
 	install -m 644 config/columbia.map $(confdir)/columbia.map
 	install -m 644 config/AMROffice.map $(confdir)/AMROffice.map
@@ -398,8 +398,8 @@
 	install -m 644 INSTALL.txt $(docdir)/INSTALL.txt
 	install -m 644 COPYING $(docdir)/COPYING
 	install -m 644 version.txt $(docdir)/version.txt
-	install -m 755 gdbhelper $(bindir)/gdbhelper
-	ln -s -f $(installed_bindir)/MobileSim $(sysbindir)/MobileSim
+	install -m 755 gdbhelper $(confdir)/gdbhelper
+
 
 uninstall:
 	-rm $(bindir)/MobileSim $(confdir)/icon.png \
