diff --git i/lib/Biber/Input/file/biblatexml.pm w/lib/Biber/Input/file/biblatexml.pm
index c2724e2..528539c 100644
--- i/lib/Biber/Input/file/biblatexml.pm
+++ w/lib/Biber/Input/file/biblatexml.pm
@@ -81,13 +81,6 @@ sub extract_entries {
       # use IO::Socket::SSL qw(debug4); # useful for debugging SSL issues
       # We have to explicitly set the cert path because otherwise the https module
       # can't find the .pem when PAR::Packer'ed
-      if (not exists($ENV{PERL_LWP_SSL_CA_FILE})) {
-        require Mozilla::CA; # Have to explicitly require this here to get it into %INC below
-        # we assume that the default CA file is in .../Mozilla/CA/cacert.pem
-        (my $vol, my $dir, undef) = File::Spec->splitpath( $INC{"Mozilla/CA.pm"} );
-        $dir =~ s/\/$//; # splitpath sometimes leaves a trailing '/'
-        $ENV{PERL_LWP_SSL_CA_FILE} = File::Spec->catpath($vol, "$dir/CA", 'cacert.pem');
-      }
       if (defined(Biber::Config->getoption('ssl-noverify-host'))) {
           $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;
       }
diff --git i/lib/Biber/Input/file/bibtex.pm w/lib/Biber/Input/file/bibtex.pm
index 2b41e7a..d581d48 100644
--- i/lib/Biber/Input/file/bibtex.pm
+++ w/lib/Biber/Input/file/bibtex.pm
@@ -118,13 +118,6 @@ sub extract_entries {
       # use IO::Socket::SSL qw(debug99); # useful for debugging SSL issues
       # We have to explicitly set the cert path because otherwise the https module
       # can't find the .pem when PAR::Packer'ed
-      if (not exists($ENV{PERL_LWP_SSL_CA_FILE})) {
-        require Mozilla::CA; # Have to explicitly require this here to get it into %INC below
-        # we assume that the default CA file is in .../Mozilla/CA/cacert.pem
-        (my $vol, my $dir, undef) = File::Spec->splitpath( $INC{"Mozilla/CA.pm"} );
-        $dir =~ s/\/$//; # splitpath sometimes leaves a trailing '/'
-        $ENV{PERL_LWP_SSL_CA_FILE} = File::Spec->catpath($vol, "$dir/CA", 'cacert.pem');
-      }
       if (defined(Biber::Config->getoption('ssl-noverify-host'))) {
           $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;
       }
diff --git i/lib/Biber/Input/file/endnotexml.pm w/lib/Biber/Input/file/endnotexml.pm
index f2f98f1..8cf6fc7 100644
--- i/lib/Biber/Input/file/endnotexml.pm
+++ w/lib/Biber/Input/file/endnotexml.pm
@@ -94,13 +94,6 @@ sub extract_entries {
       # use IO::Socket::SSL qw(debug4); # useful for debugging SSL issues
       # We have to explicitly set the cert path because otherwise the https module
       # can't find the .pem when PAR::Packer'ed
-      if (not exists($ENV{PERL_LWP_SSL_CA_FILE})) {
-        require Mozilla::CA; # Have to explicitly require this here to get it into %INC below
-        # we assume that the default CA file is in .../Mozilla/CA/cacert.pem
-        (my $vol, my $dir, undef) = File::Spec->splitpath( $INC{"Mozilla/CA.pm"} );
-        $dir =~ s/\/$//; # splitpath sometimes leaves a trailing '/'
-        $ENV{PERL_LWP_SSL_CA_FILE} = File::Spec->catpath($vol, "$dir/CA", 'cacert.pem');
-      }
       if (defined(Biber::Config->getoption('ssl-noverify-host'))) {
           $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;
       }
diff --git i/lib/Biber/Input/file/ris.pm w/lib/Biber/Input/file/ris.pm
index 6f625cb..941d3b6 100644
--- i/lib/Biber/Input/file/ris.pm
+++ w/lib/Biber/Input/file/ris.pm
@@ -90,13 +90,6 @@ sub extract_entries {
       # use IO::Socket::SSL qw(debug4); # useful for debugging SSL issues
       # We have to explicitly set the cert path because otherwise the https module
       # can't find the .pem when PAR::Packer'ed
-      if (not exists($ENV{PERL_LWP_SSL_CA_FILE})) {
-        require Mozilla::CA; # Have to explicitly require this here to get it into %INC below
-        # we assume that the default CA file is in .../Mozilla/CA/cacert.pem
-        (my $vol, my $dir, undef) = File::Spec->splitpath( $INC{"Mozilla/CA.pm"} );
-        $dir =~ s/\/$//; # splitpath sometimes leaves a trailing '/'
-        $ENV{PERL_LWP_SSL_CA_FILE} = File::Spec->catpath($vol, "$dir/CA", 'cacert.pem');
-      }
       if (defined(Biber::Config->getoption('ssl-noverify-host'))) {
           $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;
       }
diff --git i/lib/Biber/Input/file/zoterordfxml.pm w/lib/Biber/Input/file/zoterordfxml.pm
index 23e7476..1fa6b85 100644
--- i/lib/Biber/Input/file/zoterordfxml.pm
+++ w/lib/Biber/Input/file/zoterordfxml.pm
@@ -106,13 +106,6 @@ sub extract_entries {
       # use IO::Socket::SSL qw(debug4); # useful for debugging SSL issues
       # We have to explicitly set the cert path because otherwise the https module
       # can't find the .pem when PAR::Packer'ed
-      if (not exists($ENV{PERL_LWP_SSL_CA_FILE})) {
-        require Mozilla::CA; # Have to explicitly require this here to get it into %INC below
-        # we assume that the default CA file is in .../Mozilla/CA/cacert.pem
-        (my $vol, my $dir, undef) = File::Spec->splitpath( $INC{"Mozilla/CA.pm"} );
-        $dir =~ s/\/$//; # splitpath sometimes leaves a trailing '/'
-        $ENV{PERL_LWP_SSL_CA_FILE} = File::Spec->catpath($vol, "$dir/CA", 'cacert.pem');
-      }
       if (defined(Biber::Config->getoption('ssl-noverify-host'))) {
           $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;
       }
