From 747a241892b1deb8ba12216ac23656dea8b4400d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Manuel=20R=C3=BCger?= <manuel@rueg.eu>
Date: Tue, 2 Feb 2016 19:42:07 +0100
Subject: [PATCH] Fix cmake build on Linux with yubikey lib

Commit e12b2f64717efa18589784f78894303ebaedf2a5 included a typo that resulted in a failing build of pwsafe with Yubikey support on Linux.
---
 src/os/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/os/CMakeLists.txt b/src/os/CMakeLists.txt
index 652bd47..a5597b0 100644
--- a/src/os/CMakeLists.txt
+++ b/src/os/CMakeLists.txt
@@ -72,7 +72,7 @@ else()
 endif()
 
 if(HAVE_YKPERS_H)
-    list(APPEND OS_SRC ${OS_YUBI_SRC})
+    list(APPEND OS_SRCS ${OS_YUBI_SRC})
 endif()
 
 add_library(os ${OS_SRCS})
