From e12b2f64717efa18589784f78894303ebaedf2a5 Mon Sep 17 00:00:00 2001
From: Vladimir Timofeev <vovkasm@gmail.com>
Date: Fri, 29 Jan 2016 02:44:12 +0300
Subject: [PATCH] Fix cmake build on Linux without yubikey lib (NO_YUBI option)

---
 src/os/CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/os/CMakeLists.txt b/src/os/CMakeLists.txt
index dfdcd6a..652bd47 100644
--- a/src/os/CMakeLists.txt
+++ b/src/os/CMakeLists.txt
@@ -46,7 +46,6 @@ if (WIN32)
   elseif (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
   set (OS_SRCS
     linux/KeySend.cpp
-    linux/PWYubi.cpp
     linux/UUID.cpp
     linux/debug.cpp
     linux/dir.cpp
@@ -65,10 +64,15 @@ if (WIN32)
     linux/utf8conv.cpp
     linux/xsendstring.cpp
     )
+  set (OS_YUBI_SRC linux/PWYubi.cpp)
 else()
   message (FATAL_ERROR "Unsupported OS "
     ${CMAKE_SYSTEM_NAME}
     " - can't build OS library")
 endif()
 
+if(HAVE_YKPERS_H)
+    list(APPEND OS_SRC ${OS_YUBI_SRC})
+endif()
+
 add_library(os ${OS_SRCS})
